src/main/java/core/basesyntax/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.figure.type.Circle;
+import core.basesyntax.figure.type.Figures;
+import core.basesyntax.figure.type.IsoscelesTrapezoid;
+import core.basesyntax.figure.type.Rectangle;
+import core.basesyntax.figure.type.RightTriangle;
+import core.basesyntax.figure.type.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColoreSupplier coloreSupplier = new ColoreSupplier();
+
+    public Figures getRandomFigure() {
+        int figureIndex = new Random().nextInt(4);
+        switch (figureIndex) {
+            case 0 :
+                return new Circle(coloreSupplier.getRandomColor(), getRandomInt());
```

Comment:
```suggestion
                return getRandomCircle();
```
make methods that create each random figure